src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -468,9 +474,9 @@ class MockExecutionServiceDAO() extends ExecutionServiceDAO {
   }
 
   override def logs(id: String, userInfo: UserInfo) = Future.successful(ExecutionServiceLogs(id,
-    Map("x" -> Seq(Map(
-      "stdout" -> "gs://cromwell-dev/cromwell-executions/wf/this_workflow_exists/call-x/job.stdout.txt",
-      "stdout" -> "gs://cromwell-dev/cromwell-executions/wf/this_workflow_exists/call-x/job.stdout.txt")))))
```

Comment:
This doesn't make sense, so I assume it was never used anywhere.  I updated it anyway.